workspaces/ui/src/components/diff/v2/Finalize.js

```diff
@@ -76,7 +71,18 @@ export default function FinalizeDialog(props) {
       JsonHelper.jsArrayToVector([EndBatchCommit(batchId)])
     );
     await specService.saveEvents(newEventStore, rfcId);
-    setTimeout(() => history.push(`${baseUrl}/diffs/${captureId}`), 500);
+    
+    setTimeout(() => {
```

Comment:
I'm curious why this was delayed to begin with

Reply:
no idea, it was here before I started (doesn't seem needed though) — when I remove it, core functionality seems unaffected

Reply:
went ahead and removed the timeout for `history.push` for now